smalltalksrc/Slang/CCodeGenerator.class.st

```diff
@@ -4731,12 +4765,11 @@ CCodeGenerator >> staticallyResolveMethodNamed: selector forClass: aClass to: st
 	 prefixed with a given string to disambiguate. This hack allows us to use two different
 	 compaction algorithms with the same API at the same time; the selection being done
 	 by a class which holds the flag stating which algorithm is in effect at the current time."
+
 	| method |
-	method := methods
-					removeKey: selector
-					ifAbsent:
-						[self logger cr; nextPutAll: 'warning: did not find ', selector, ' to be able to map to ', staticallyResolvedSelector.
-						^self].
+
+1halt.
```

Comment:
Do we keep the halt here?

Reply:
Usually should not arrive here... It is building and working, so sounds as dead code to me